### PR TITLE
Bottom panel

### PR DESCRIPTION
**Pull request by elacymerys**

### CODE REVIEW

**Review by MaksymilianK — DISMISSED** [review]
Nice work!

**Review by MaksymilianK — DISMISSED** [review]

File: thesis-gui/src/context/CategoryContext.tsx [hunk 1]
```diff
@@ -38,6 +38,8 @@ export const CategoryContextProvider: FC = ({ children }) => {
             getCategories();
         } else {
             setChosenCategories([]);
+        } else {
```
Comment (MaksymilianK) [inline thesis-gui/src/context/CategoryContext.tsx]: double **else**

**Review by MaksymilianK — CHANGES_REQUESTED** [review]

File: thesis-gui/src/context/CategoryContext.tsx (hunk 1 above)
Comment (elacymerys) [reply thesis-gui/src/context/CategoryContext.tsx]: A, yes, it happens after merge. I will fix it

Comment (elacymerys) [reply thesis-gui/src/context/CategoryContext.tsx]: Fixed

**Review by MaksymilianK — DISMISSED** [review]

**Review by Lozoko — APPROVED** [review]

**Pull request merged**